docs/developers/SDK/technical-references/sdk-intro.md

```diff
@@ -0,0 +1,118 @@
+---
+sidebar_position: 0
+---
+
+# SDK Introduction
+
+:::info **Overrides**
+
+All API functions that produce a signed transaction can be equipped with the usual `ethers.js` overrides as optional parameters.
+
+:::
+
+## Mangrove
+
+The root class of the API. Use `Mangrove.connect` to get an instance of it. Here are a few possibilities:
+
+```typescript
+mgv = await Mangrove.connect(window.ethereum); // web browser
+mgv = await Mangrove.connect('http://127.0.0.1:8545'); // HTTP provider
+mgv = await Mangrove.connect(); // Uses Ethers.js fallback mainnet (for testing only)
+mgv = await Mangrove.connect('rinkeby'); // Uses Ethers.js fallback (for testing only)
+// Init with private key (server side)
+mgv = await Mangrove.connect(
+'https://mainnet.infura.io/v3/_your_project_id_', // provider
+{
+  privateKey: '0x_your_private_key_', // preferably with environment variable
+});
+// Init with HD mnemonic (server side)
+mgv = await Mangrove.connect( {
+  signer: myEthersWallet
+});
+```
+
+You can test you are indeed connected to the [deployed Mangrove](../../addresses/contract-addresses.md) by asking for the current global configuration of Mangrove:
+
+`config = await mgv.config()`
+
+The `mgv` object gives you access to a variety of useful properties, such as `Token`, `Market` and `OfferLogic` (allowing one to connect to an on-chain offer logic).
+
+:::info
+
+`mgv.contract` gives access to the underlying `ethers.js` contract object, and allows you to interact with the deployed `Mangrove` using low-level `ethers.js` calls. Hence, 
+
+```typescript
+await mgv.contract.f(...)
+```
+
+will execute the `ethers.js` call to the function `f` on the Mangrove contract (signed when needed by the `signer` provided to the `connect` function).
+
+:::
+
+## Token
+
+This class provides easy means to interact with a deployed contract on the standard [EIP-20](https://eips.ethereum.org/EIPS/eip-20). To obtain an instance use:
+
+```typescript
+mgvTkn = await mgv.token("<tokenSymbol>"); // e.g "DAI", "WETH", "amDAI", etc.
```

Comment:
Since `context-addresses` is mentioned below, maybe mention that a `tokenId` is also an option here?
And perhaps mention that tokens can also be instantiated from their address via `tokenFromAddress`.

Reply:
Taken into account in 4f7d8c7f5e1b618704d6c738a348104a4cf7ef48